### PR TITLE
replaces instances of VMware datacenter with two words

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.16+

Issue:
N/A

Link to docs preview:
- [Understanding image registry repository mirroring](https://78317--ocpdocs-pr.netlify.app/openshift-dedicated/latest/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration) (dedicated)
- [Understanding image registry repository mirroring](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration) (OCP)
- [Understanding image registry repository mirroring](https://78317--ocpdocs-pr.netlify.app/openshift-rosa/latest/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration) (ROSA)
- [Additional VMware vSphere configuration parameters](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installation-config-parameters-vsphere.html#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere)
- [VMware vSphere region and zone enablement](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations#installation-vsphere-regions-zones_installing-vsphere-installer-provisioned-customizations)
- [Creating the installation configuration file](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations#installation-initializing_installing-vsphere-installer-provisioned-customizations)
- [Sample YAML for a compute machine set custom resource on vSphere](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-vsphere#machineset-yaml-vsphere_creating-machineset-vsphere)
- [Sample VMware vSphere provider specification](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-vsphere#cpmso-yaml-provider-spec-vsphere_cpmso-config-options-vsphere)
- [Sample YAML for a Cluster API machine template resource on VMware vSphere](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/cluster_api_machine_management/cluster_api_provider_configurations/cluster-api-config-options-vsphere#capi-yaml-machine-template-vsphere_cluster-api-config-options-vsphere)
- [Configuring the vSphere connection settings](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/installing-vsphere-post-installation-configuration#configuring-vSphere-connection-settings_installing-vsphere-post-installation-configuration)
- [vSphere CSI topology overview](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#persistent-storage-csi-vsphere-top-aware-overview_persistent-storage-csi-vsphere)
- [vSphere CSI topology requirements](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#vsphere-csi-topology-requirements)
- [Sample YAML for a Windows MachineSet object on vSphere](https://78317--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#windows-machineset-vsphere_creating-windows-machineset-vsphere)

QE review:
N/A (terms compliance)

Additional information:

- Based on [VMware docs](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-7FDFBDBE-F8AC-4D00-AE5E-3F14D7472FAF.html) usage
- "datastore" is [one word but lowercase](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-D5AB2BAD-C69A-4B8D-B468-25D86B8D39CE.html)
- Azure seems to use "datacenter" so only changed VMware instances

### CODE REVIEW

File: modules/capi-yaml-machine-template-vsphere.adoc [hunk 1]
```diff
@@ -51,8 +51,8 @@ The following values are valid:
 +
 When using the `linkedClone` type, the disk size matches the clone source instead of using the `diskGiB` value.
 For more information, see the {vmw-short} documentation about VM clone types.
-<7> Specify the vCenter Datacenter to deploy the compute machine set on.
-<8> Specify the vCenter Datastore to deploy the compute machine set on.
+<7> Specify the vCenter data center to deploy the compute machine set on.
+<8> Specify the vCenter datastore to deploy the compute machine set on.
```
Comment (mburke5678) [inline modules/capi-yaml-machine-template-vsphere.adoc:55]: ISG uses "data store", but our doc seem to prefer "datastore". Shrug.

```suggestion
<8> Specify the vCenter data store to deploy the compute machine set on.
```

File: modules/cpmso-yaml-provider-spec-vsphere.adoc [hunk 2]
```diff
@@ -69,8 +69,8 @@ If you specify this value in the provider specification when using failure domai
 If the cluster is configured to use a failure domain, these parameters are configured in the failure domain.
 If you specify these values in the provider specification when using failure domains, the Control Plane Machine Set Operator ignores them.
 ====
-<11> Specifies the vCenter Datacenter for the control plane.
-<12> Specifies the vCenter Datastore for the control plane.
+<11> Specifies the vCenter data center for the control plane.
+<12> Specifies the vCenter datastore for the control plane.
```
Comment (mburke5678) [inline modules/cpmso-yaml-provider-spec-vsphere.adoc:73]: ??

```suggestion
<12> Specifies the vCenter data store for the control plane.
```

File: modules/installation-initializing.adoc [hunk 3]
```diff
@@ -333,7 +333,7 @@ The installation program connects to your vCenter instance.
 +
 [NOTE]
 ====
-After you create the installation configuration file, you can modify the file to create a multiple vSphere datacenters environment. This means that you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
+After you create the installation configuration file, you can modify the file to create a multiple vSphere data centers environment. This means that you can deploy an {product-title} cluster to multiple vSphere data centers that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```
Comment (mburke5678) [inline modules/installation-initializing.adoc:336]: Beyond the scope, but.....

```suggestion
After you create the installation configuration file, you can modify the file to create a multiple vSphere data centers environment. This means that you can deploy an {product-title} cluster to multiple vSphere data centers that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```
```suggestion
After you create the installation configuration file, you can modify the file to create a multiple vSphere data center environment. This means that you can deploy an {product-title} cluster to multiple vSphere data centers that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 4]
```diff
@@ -81,7 +81,7 @@ endif::upi[]
 `VApp.Import`
 `VirtualMachine.Config.AddNewDisk`
 
-|vSphere Datastore
+|vSphere datastore
```
Comment (mburke5678) [inline modules/installation-vsphere-installer-infra-requirements.adoc:84]: ?
```suggestion
|vSphere data store
```

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 5]
```diff
@@ -212,7 +212,7 @@ endif::upi[]
 `VApp.Import`
 `"Virtual machine"."Change Configuration"."Add new disk"`
 
-|vSphere Datastore
+|vSphere datastore
```
Comment (mburke5678) [inline modules/installation-vsphere-installer-infra-requirements.adoc:215]: ?
```suggestion
|vSphere data store
``

**Comment by mburke5678** [comment]
@jeana-redhat A few things. Otherwise LGTM

File: modules/capi-yaml-machine-template-vsphere.adoc (hunk 1 above)
Comment (jeana-redhat) [reply modules/capi-yaml-machine-template-vsphere.adoc:55]: this is specifically for VMware, and they have "[datastore](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-D5AB2BAD-C69A-4B8D-B468-25D86B8D39CE.html)" as a single word. I agree elsewhere/generically it should be two words though

File: modules/installation-initializing.adoc (hunk 3 above)
Comment (jeana-redhat) [reply modules/installation-initializing.adoc:336]: not at all! might as well fix it :nerd_face: thanks!

**Pull request merged**